km51/Beshta_Vladyslav/laboratory1.sql

```diff
@@ -1,2 +1,52 @@
 -- LABORATORY WORK 1
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+змінювати структуру таблиць.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER Beshta IDENTIFIED BY beshta
+DEFAULT TABLESPACE "USERS"
+TEMPORARY TABLESPACE "TEMP";
+
+ALTER USER Beshta QUOTA 100M ON USER;
+
+GRANT "CONNECT" TO BESHTA;
+
+GRANT ALTER ANY TABLE TO Beshta
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Книжка складається з сторінок, на яких є рядки тексту.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+CREATE TABLE BOOK(
+book_name VARCHAR(30) NOT NULL
+);
+
+ALTER TABLE BOOK ADD CONSTRAINT pk_book_name PRIMARY KEY(book_name);
+
+CREATE TABLE BOOK_PAGE(
```

Comment:
The book_page can not be the key. In another book there is a page with the same number. You must use book_id as a foreign key.